packages/analyticsdx-template-lint/src/schemas/auto-install-schema.json

```diff
@@ -85,7 +85,10 @@
             },
             "values": {
               "type": "object",
-              "additionalProperties": true,
+              "additionalProperties": false,
+              "patternProperties": {
+                "^[a-zA-Z_][a-zA-Z0-9_]*": {}
```

Comment:
Should there be a `$` at the end of this regex?

Reply:
Good catch on these 2 regexes, yes there should be. I'll fix it up.